### PR TITLE
cmd/geth: don't repeat yourself

### PR DESCRIPTION
**Pull request by jsvisa**

### CODE REVIEW

File: cmd/geth/main.go [hunk 1]
```diff
@@ -287,11 +287,11 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 				status, _ := event.Wallet.Status()
 				log.Info("New wallet appeared", "url", event.Wallet.URL(), "status", status)
 
+				derivationPath := accounts.DefaultBaseDerivationPath
```
Comment (ligi) [inline cmd/geth/main.go]: great DRYing this out! Just a bit sad there is now this one useless assign in some cases - in Kotlin you can do :

```
derivationPath = if ( event.Wallet.URL().Scheme == "ledger" ) {
accounts.DefaultLedgerBaseDerivationPath
} else {
accounts.DefaultBaseDerivationPath
}
```

Comment (jsvisa) [reply cmd/geth/main.go]: Yeah, Golang don't support if else assign

**Comment by fjl** [comment]
IMHO this change doesn't improve readability at all.

**Comment by ligi** [comment]
@fjl too bad. what do you think about using a function? Think the best here would be a if else assign - but as go does not support it perhaps a function is the next best thing - like getDerivationPathForScheme ?

**Comment by jsvisa** [comment]
@ligi  agree with a util function.

**Comment by ligi** [comment]
@jsvisa Great! - let's hope @fjl also likes the idea ;-)

**Comment by fjl** [comment]
But why does it need to change at all? It's not like there is massive duplication here, the current code is just the easiest way to express this. Adding a function would make sense if we had more schemes but isn't really needed.

**Pull request merged**